### PR TITLE
HACCP-6: Progress bar (amends).

### PR DESCRIPTION
**Pull request by beloglazov91**

### CODE REVIEW

File: src/components/haccp/WebToolProgressBar/webToolProgressBar.html.twig [hunk 1]
```diff
@@ -7,7 +7,9 @@
       <div class="webtool-progress-bar__steps">
         {% for step in line.steps %}
           <div class="webtool-progress-bar__step webtool-progress-bar__step-{{ step.state }}">
-            <a href="{{ step.url }}" class="webtool-progress-bar__step--{{ line.label|lower }}-{{ step.label|lower }}" onClick="{{ step.callback ? step.callback ~ '(); return false;' : '' }}">{{ step.label }}</a>
+            <a href="{{ step.url is defined ? step.url : '#' }}" class="webtool-progress-bar__step--{{ line.label|lower }}-{{ step.label|lower }}" onClick="{{ step.callback is defined ? step.callback ~ '(); return false;' : '' }}">
```
Comment (c-ancia) [inline src/components/haccp/WebToolProgressBar/webToolProgressBar.html.twig:10]: why did you need to add `href="{{ step.url is defined ? step.url : '#' }}"` please? 🤔

File: src/components/haccp/WebToolProgressBar/webToolProgressBar.scss [hunk 2]
```diff
@@ -113,9 +129,14 @@
...
+      &:hover {
+        background: var(--fsa-primary_dark-green);
+        color: var(--fsa-primary_white);
+      }
+
       &:focus {
-        color: var(--fsa__link-green);
+        background: var(--fsa-tertiary_light-yellow);
+        color: var(--fsa-primary_black);
       }
```
Comment (c-ancia) [inline src/components/haccp/WebToolProgressBar/webToolProgressBar.scss:140]: any reason why you had to specify the hover & focus state instead of using the `@include link;` mixin please? 🤔

File: src/components/haccp/WebToolProgressBar/webToolProgressBar.html.twig (hunk 1 above)
Comment (beloglazov91) [reply src/components/haccp/WebToolProgressBar/webToolProgressBar.html.twig:10]: Because sometime url will be absent, for the cases when we will use callbacks. In this case URL is not required to pass. But better fill it with "#" than leave as empty.

**Comment by beloglazov91** [comment]
@c-ancia Check it again please.

File: src/components/haccp/WebToolProgressBar/webToolProgressBar.html.twig (hunk 1 above)
Comment (c-ancia) [reply src/components/haccp/WebToolProgressBar/webToolProgressBar.html.twig:10]: okay, I saw it everywhere in the story so that's why I was wondering ^^

**Review by c-ancia — APPROVED** [review]
